File: base/templates/java/Controller.java
${packageName};

import io.micronaut.http.annotation.Controller
import io.micronaut.http.annotation.Get
import io.micronaut.http.HttpStatus

@Controller("/${propertyName}")
public class ${className}Controller {

    @Get("/")
    public HttpStatus index() {
        return HttpStatus.OK
    }
}

File: service/templates/kotlin/Controller.kt
${packageName ? 'package ' + packageName : '' }

import io.micronaut.http.annotation.Controller
import io.micronaut.http.annotation.Get
import io.micronaut.http.HttpStatus

@Controller("/${propertyName}")
class ${className} {

    @Get("/")
    fun index(): HttpStatus {
        return HttpStatus.OK
    }
}